packages/p2p/src/components/buy-sell/rate-change-modal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Modal, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize, Localize } from 'Components/i18next';
+import 'Components/buy-sell/rate-change-modal.scss';
+
+const RateChangeModal = ({ local_currency, should_show_rate_change_modal, setShouldShowRatechangeModal }) => {
+    const closeModal = () => {
+        setShouldShowRatechangeModal(false);
+    };
+    return (
+        <Modal is_open={should_show_rate_change_modal} toggleModal={closeModal} width={'440px'}>
```

Comment:
A question: if we use just `width=440`, will it work? Because, normally we do not need to specify the unit.

Reply:
I think the unix of measurement is required as `width` property is binded with `style`.  I will replace with the `small` prop. That is doing the exact same thing